catalog/core/catalog-core-standardframework/src/main/java/ddf/catalog/history/Historian.java

```diff
@@ -284,6 +285,27 @@ public UpdateResponse versionUpdate(UpdateResponse updateResponse)
     }
 
     /**
+     * Versions deleted {@link Metacard}s.
+     * @param deleteResponse Versions this responses deleted metacards
+     */
+    public void versionDelete(DeleteResponse deleteResponse) {
```

Comment:
Its following the naming convention of the other metacards. In reality now I think bout it, since they are all differentiated by their method params, we could just name them all `version` (since the parameter declares what it is versioning). How does that sound?